cmd/ci-operator-checkconfig/main.go

```diff
@@ -88,3 +89,16 @@ func validateTags(seen tagSet) []error {
 	}
 	return dupes
 }
+
+func main() {
+	o := options{}
+	if err := o.parse(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+	}
+	if errs := o.validate(); errs != nil {
+		fmt.Fprintln(os.Stderr, "error validating configuration files:")
```

Comment:
might as well use `logrus`? this is like our one binary that doesn't, which is weird